simapp/app.go

```diff
@@ -241,7 +242,7 @@ func NewSimApp(
 		voteExtHandler := NewVoteExtensionHandler()
 		voteExtHandler.SetHandlers(bApp)
 	}
-	baseAppOptions = append(baseAppOptions, voteExtOp)
+	baseAppOptions = append(baseAppOptions, voteExtOp, baseapp.SetOptimisticExecution())
```

Comment:
Is it purposely done so that simapp only uses optimistic execution?
Or should we add it here: https://github.com/cosmos/cosmos-sdk/blob/e2637b817ca96263f9e01a22631abc31cd289797/server/util.go#L529?

Reply:
So I wouldn't like to add a flag for this, because that could give node operators the chance of not running optimistic execution if the developers chose to do so (they can still modify the code but that takes some extra effort).
Could we add a flag on simapp only? wdyt?

Reply:
Yeah, a flag wouldn't make sense.
I was taking about adding it in our default baseapp options.
But I think I remember conversation about having it disabled by default first 🤔